platform/security/platform-security-core-api/src/main/java/ddf/security/samlp/SimpleSign.java

```diff
@@ -248,19 +274,13 @@ public boolean validateSignature(
           certificateFactory.generateCertificate(
               new ByteArrayInputStream(Base64.getMimeDecoder().decode(encodedPublicKey)));
 
-      String jceSigAlgo = "SHA1withRSA";
-      if ("DSA".equalsIgnoreCase(certificate.getPublicKey().getAlgorithm())) {
-        jceSigAlgo = "SHA1withDSA";
-      }
+      java.security.Signature sig = getSignature((X509Certificate) certificate);
 
-      java.security.Signature sig = java.security.Signature.getInstance(jceSigAlgo);
```

Comment:
❗️ Technically this is out of scope of this PR since it was already broken, but we need to pull the SigAlg from the `queryParamsToValidate`. That has this SigAlg in it. So this code should read something like:
```
String jceSigAlgo = <parse SigAlg parameter from queryParamsToValidate>
java.security.Signature sig = java.security.Signature.getInstance(jceSigAlgo);
```